src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -4,5 +4,13 @@
  * Feel free to remove this class and create your own.
  */
 public class HelloWorld {
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
 
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[10];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+            figures[i].draw();
+        }
```

Comment:
![image](https://user-images.githubusercontent.com/81781479/187749938-d0a5b350-edcf-41b2-af7e-5d1b82d4ef26.png)
half of figures should be random, another half default (default circle)